package.json

```diff
@@ -45,7 +45,7 @@
     "ethers": "^5.0.8",
     "is-ipfs": "^2.0.0",
     "prop-types": "^15.7.2",
-    "react": "^16.13.1",
+    "react": "^16.12.0",
```

Comment:
Not sure what happened with this version change, but we should keep the current :)